src/viewmodel/prototype/get.js

```diff
@@ -62,6 +63,12 @@ export default function Viewmodel$get ( keypath, options ) {
 		value = wrapped.get();
 	}
 
+	if ( keypath.isRoot && options.fullRootGet ) {
+		for ( key in this.mappings ) {
+			value[ key ] = this.mappings[ key ].getValue();
+		}
```

Comment:
Nothing great has come to me on this. We could modify the magic adaptor so it recognizes the `''` keypath and creates wrappers for mappings. Then everything works as expected for magic mode.

Non-magic mode is a headache because the non-object mapping values are stale once we return whatever we return:

```js
// mapping 'foo' currently 'bar'
var data = ractive.get();
data.foo; // 'bar'
ractive.set('foo','qux');
data.foo; // still 'bar' :(
```

Which is also why I don't think we can append the mapping keys to the data object and need to make a copy.